lib/modules/apostrophe-utils/lib/api.js

```diff
@@ -326,14 +326,19 @@ module.exports = function(self, options) {
   // (Array.isArray returns true). Otherwise all objects with
   // a length property would be treated as arrays, which is
   // an unrealistic restriction on apostrophe doc schemas.
+  //
+  // If o itself is a string, boolean or other scalar type it is returned
+  // as-is.
 
   self.clonePermanent = function(o, keepScalars) {
     var c;
     var isArray = Array.isArray(o);
     if (isArray) {
       c = [];
-    } else {
+    } else if ((o != null) && ((typeof o) === 'object')) {
       c = {};
+    } else {
+      return o;
```

Comment:
It seems like a new effect that a null value passed in would return directly. We don't have a test for that. Should it return an empty object as before?

Reply:
I think it's a bug fix. clonePermanent should not return something totally different, it should return something that is equivalent in every way, except for the temporary properties.